.changeset/fluffy-tips-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux/ui-components": patch
+"@sap-ux/prompting-ui": patch
```

Comment:
there no public `@sap-ux/prompting-ui`  - it is private as examples ->
public is `@sap-ux/ui-prompting`

Reply:
I will create separate PR to rename examples package @sap-ux/prompting-ui to avoid mistakes